### DISCUSSION TITLE
Falcon LLM text-gen-web-UI support is more important than anything else atm, imo

### DISCUSSION THREAD

**Discussion by sarutobiumon** [oobabooga/text-generation-webui#2375] [General]
This thing is incredible and could possibly be the starting point of a much bigger leap for open source LLM's that are commercially usable.

Description and demo: https://www.youtube.com/watch?v=_odWpQ5aVUI

@oobabooga please consider, ty.

**Comment by cibernicola** [comment]
It works already, maybe not at its best, but works.
